apps/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ if (BUILD_GUI)
 	endif()
 endif()
 
+if (BUILD_PYTHON)
+    add_subdirectory (vapi)
```

Comment:
I assume everything except main.cpp in apps/vapi will moved to the lib directory?

Reply:
Yes, I will move it later.